src/i18n/en/common.json

```diff
@@ -43,6 +43,52 @@
       "screenReaderHeading": "Selected plots:"
     }
   },
+  "areaSearch": {
```

Comment:
This looks like a content that would be wise to be generated backend side.
These links and texts are propably things that someone wants to update every now and then so would be nice to edit these in DB instead of codebase.
If I remember, I bring this discussion up on daily or bi-weekly or somewhere.

Reply:
Maybe. Haven't heard a need for them to be editable on the fly, but doesn't hurt to discuss.